torch/ao/quantization/observer.py

```diff
@@ -1512,3 +1546,19 @@ def load_observer_state_dict(mod, obs_dict):
 Default observer for operators like reshape that reuses the observer of input to
 the operator
 """
+
+_FIXED_QPARAMS_OP_TO_OBSERVER: Dict[Union[Callable, str], _PartialWrapper] = {
```

Comment:
I feel this should probably live in somewhere else, it's a bit weird to put this mapping in observer.py, maybe move this to qconfig_mapping.py?